doc/modules/mod_muc_light_commands.md

```diff
@@ -0,0 +1,49 @@
+# MongooseIM's light multi-user chat commands set
```

Comment:
"light multi-user chat" or rather "multi-user chat light" like in "MUC light" and not "light MUC"?